doc/lua_api.txt

```diff
@@ -1725,7 +1725,13 @@ to games.
     * `3`: the node always gets the digging time 0 seconds (torch)
 * `disable_jump`: Player (and possibly other things) cannot jump from node
   or if their feet are in the node. Note: not supported for `new_move = false`
-* `fall_damage_add_percent`: damage speed = `speed * (1 + value/100)`
+* `fall_damage_add_percent`: modifies the fall damage suffered when hitting
+  the top of this node. The damage formula is:
+    node damage speed = `speed * (1 + value/100) * player_damage_speed`
```

Comment:
"node damage speed" is a very artificial term. I think it would make sense to refer to a separate location within `lua_api.txt` for both, node and armor damage groups.

```
The final player damage is determined by the following formula:

damage = collision speed
* ((fall_damage_add_percent + 100) / 100) node group
* ((fall_damage_add_percent + 100) / 100) player armor group
- (14) constant tolerance
```